ecs-init/exec/iptables/iptables.go

```diff
@@ -48,14 +48,15 @@ const (
 	iptablesTableFilter = "filter"
 	iptablesTableNat    = "nat"
 
-	offhostIntrospectionAccessConfigEnv   = "ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS"
-	offhostIntrospectonAccessInterfaceEnv = "ECS_OFFHOST_INTROSPECTION_INTERFACE_NAME"
-	agentIntrospectionServerPort          = "51678"
+	offhostIntrospectionAccessConfigEnv = "ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS"
+	agentIntrospectionServerPort        = "51678"
+	ipv6KernelConfigPath                = "/proc/sys/net/ipv6"
 )
 
 var (
 	defaultLoopbackInterfaceName   = ""
 	defaultDockerBridgeNetworkName = ""
+	checkFile                      = os.Stat
```

Comment:
tiny nit: `/proc/sys/net/ipv6` seems to be implemented like a directory, and therefore can consider renaming `checkFile` to something like `checkPathExists` etc